### DISCUSSION TITLE
Daily / weekly digest

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/PublishPress-Planner#1344] [Feature Requests]
>To fully cover my needs, I also need the option to set a digest for notifications (Notification frequency), as you can see in the edited screenshot, but it can be more tricky to manage for the users because I think a "repetition" area in mail templates is required.
![Screen Shot 2021-06-04 at 17 24 38-digest](https://user-images.githubusercontent.com/407131/121220725-416a5980-c85b-11eb-844e-620d2420161e.jpg)

**Comment by stevejburge** [comment]
One more request for this, referencing this option https://betternotificationsforwp.com/downloads/digest/
